settings.py

```diff
@@ -23,7 +23,12 @@ class Settings(BaseSettings):
     ] = "https://developer.sandbox.ioxio-dataspace.com/resources"
 
     DATASPACE_BASE_DOMAIN: str = "sandbox.ioxio-dataspace.com"
-    AUTHENTICATION_PROVIDER_URL: HttpUrl = "https://login.sandbox.ioxio-dataspace.com"
+    AUTHENTICATION_PROVIDER_DEVELOPER_URL: HttpUrl = (
+        "https://login.sandbox.ioxio-dataspace.com"
+    )
+    AUTHENTICATION_PROVIDER_END_USER_URL: HttpUrl = (
+        "https://login.sandbox.ioxio-dataspace.com"
+    )
```

Comment:
Is it possible to document these as "https URLs"?

Reply:
Since these now both are pointing to the same value, it makes less great example values in the end

could we instead use something like `auth.end-user.example.com` and `auth.developer.example.com`? or similar? Is it important that these say "sandbox"?

Reply:
As discussed elsewhere agreed to not do any of these changes at the current time.